peekingduck/declarative_loader.py

```diff
@@ -198,3 +197,32 @@ def get_pipeline(self) -> Pipeline:
         except ValueError as error:
             self.logger.error(str(error))
             sys.exit(1)
+
+
+class NodeList:
```

Comment:
I think it's okay to keep `NodeList` class within declarative_loader.py. It isn't used elsewhere except `runner.py` and even then it's only for typehinting.